_maps/tannhauser/automapper/automapper_config.toml

```diff
@@ -0,0 +1,73 @@
+# Automapper configuration
```

Comment:
from what i read of the automapper code, the auto mapper only can load a single toml file and its coded to point at the skyrat one, so is this file even worth having?

Reply:
Its more a copy of our changes and a place hold for if we can have it load a 2nd toml file so we dont have to edit the skyrat one.

Will look at the code later to see if we can add a 2nd toml on a new branch